blue_bringup/config/controllers.xml

```diff
@@ -22,17 +22,19 @@
 
     gripper_controller:
       type: effort_controllers/GripperActionController
-      joint:  $(arg side)_gripper_joint
+      joints:
+        - $(arg side)_gripper_joint
+      joint: $(arg side)_gripper_joint
       goal_tolerance: 0.01
       stalled_velocity_threshold: 0.01
       stall_timeout: 0.2
       gains:
-        $(arg side)_gripper_joint: {p: 15, i: 5, d: 10.5, i_clamp: 1}
+        $(arg side)_gripper_joint: {p: 15, i: 5, d: 10.5, i_clamp_min: -1, i_clamp_max: 1}
 
     ### Joint controllers
 
     joint_torque_controller:
-      type: effort_controllers/JointGroupEffortController
+      type: effort_controllers/JointTrajectoryController #JointGroupEffortController
```

Comment:
Was there a specific reason for removing the joint torque controller? Would it make sense to create a separate controller for the trajectory controller?

Reply:
The MoveIt move_group interface publishes a FollowJointTrajectory goal which is handled by a JointTrajectoryController from ros_control.

I can create another controller though, I'll go change some names around.

Reply:
Ok, all relevant changes to this should be reflected across all repos.